src/Header/RequestHandler.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "./OperateData.h"
+#include "./Authenticator.h"
+#include "./APIHandler.h"
+
+//Will decode the request string and check if it's a valid request.
+//calling RequestParser to get OperateData then call correspond api function
+class RequestHandler
+{
+
+public:
+    //default constructor will never be used
+    RequestHandler();
+    //instantiate instance with request string, will parse to OperateData and check with Authenticator and call correspond api.
+    RequestHandler(const string &requestData);
+    //data is content of request body
+    unique_ptr<OperateData> data;
```

Comment:
You need to include <memory> for this. Also, reference it as std::unique_ptr.

Reply:
Done